### PR TITLE
Two tests for showing errors

### PR DESCRIPTION
**Pull request by kshyatt**

### CODE REVIEW

File: test/errorshow.jl [hunk 1]
```diff
@@ -277,6 +280,11 @@ let undefvar
     @test err_str == "ArgumentError: not an error"
     err_str = @except_str @assert(false) AssertionError
     @test err_str == "AssertionError: false"
+
+    err_str = @except_str WString[] UndefVarError
+    @test err_str == """`WString` has been moved to the package LegacyStrings.jl:
```
Comment (JeffBezanson) [inline test/errorshow.jl]: These were moved in 0.5, so maybe the code for this should just be removed?

Comment (kshyatt) [reply test/errorshow.jl]: Do we want to keep these tests/lines? I need an adult!

Comment (KristofferC) [reply test/errorshow.jl]: Personally, I don't think these are needed / that useful.

**Comment by kshyatt** [comment]
Nearly a year after the fact I updated this PR in response to feedback <3

**Comment by Keno** [comment]
I guess this is still relevant. Re-running CI.

**Comment by KristofferC** [comment]
I think this can be closed since there was so long since `String` had a data field.

**Pull request closed without merging**